### PR TITLE
update implementers and verifier websites

### PR DESCRIPTION
**Pull request by Sakurann**
@gauravping, @jischr, @awoie, @troyronda please add urls to your verifiers that people can test against if you plan to maintain them.

### CODE REVIEW

**Comment by justAnIdentity** [comment]
@Sakurann  url to spruce verifier: https://app.vp.interop.spruceid.xyz/

File: spec/spec.md [hunk 1]
```diff
@@ -682,12 +682,19 @@ Examples are listed inline in above sections as well as in complete form within
...
+- Auth0Lab
 
 ## Testing
 
-Implementations may test conformance of the wallets to this profile using this [verification website](https://vcinteroptesting.azurewebsites.net/verifier).
+Implementations may test conformance of the wallets to this profile using the following verification websites:
+
+- https://vcinteroptesting.azurewebsites.net/verifier
+- https://verifiablecredentials.dev/presentation
+
```
Comment (awoie) [inline spec/spec.md:696]:
```suggestion
https://app.vp.interop.spruceid.xyz/

```

**Review by awoie — CHANGES_REQUESTED** [review]
pls add spruce verifier as suggested

File: spec/spec.md (hunk 1 above)
Comment (justAnIdentity) [reply spec/spec.md:697]: could use a dash so it's part of the list

File: spec/spec.md [hunk 2]
```diff
@@ -682,12 +682,20 @@ Examples are listed inline in above sections as well as in complete form within
...
+- Auth0Lab
 
 ## Testing
 
-Implementations may test conformance of the wallets to this profile using this [verification website](https://vcinteroptesting.azurewebsites.net/verifier).
+Implementations may test conformance of the wallets to this profile using the following verification websites:
+
+- https://vcinteroptesting.azurewebsites.net/verifier
+- https://verifiablecredentials.dev/presentation
+https://app.vp.interop.spruceid.xyz/
```
Comment (jischr) [inline spec/spec.md:696]:
```suggestion
- https://app.vp.interop.spruceid.xyz/
```

**Review by jischr — APPROVED** [review]

**Comment by gauravping** [comment]
Please add Ping Identity verifier at https://verifycredential.pingone.com/verifier.html

**Comment by sudeshrshetty** [comment]
@Sakurann please add Gen verifier https://demo-rp.stg.trustbloc.dev/verifierqr

File: spec/spec.md [hunk 3]
```diff
@@ -682,12 +682,20 @@ Examples are listed inline in above sections as well as in complete form within
...
 
 ## Testing
 
-Implementations may test conformance of the wallets to this profile using this [verification website](https://vcinteroptesting.azurewebsites.net/verifier).
+Implementations may test conformance of the wallets to this profile using the following verification websites:
+
+- https://vcinteroptesting.azurewebsites.net/verifier
+- https://verifiablecredentials.dev/presentation
+- https://app.vp.interop.spruceid.xyz/
+
```
Comment (jischr) [inline spec/spec.md:697]:
```suggestion
https://verifycredential.pingone.com/verifier.html
https://demo-rp.stg.trustbloc.dev/verifierqr

```

File: spec/spec.md [hunk 4]
```diff
@@ -682,12 +682,22 @@ Examples are listed inline in above sections as well as in complete form within
...
 ## Testing
 
-Implementations may test conformance of the wallets to this profile using this [verification website](https://vcinteroptesting.azurewebsites.net/verifier).
+Implementations may test conformance of the wallets to this profile using the following verification websites:
+
+- https://vcinteroptesting.azurewebsites.net/verifier
+- https://verifiablecredentials.dev/presentation
+- https://app.vp.interop.spruceid.xyz/
+https://verifycredential.pingone.com/verifier.html
+https://demo-rp.stg.trustbloc.dev/verifierqr
```
Comment (Sakurann) [inline spec/spec.md:698]:
```suggestion
- https://vcinteroptesting.azurewebsites.net/verifier
- https://app.vp.interop.spruceid.xyz/
- https://verifycredential.pingone.com/verifier.html
- https://demo-rp.stg.trustbloc.dev/verifierqr
- https://verifiablecredentials.dev/presentation
```

File: spec/spec.md [hunk 5]
```diff
@@ -682,12 +682,22 @@ Examples are listed inline in above sections as well as in complete form within
 
 - Microsoft
 - Workday
```
Comment (Sakurann) [inline spec/spec.md:684]:
```suggestion
```

File: spec/spec.md [hunk 6]
```diff
@@ -682,12 +682,22 @@ Examples are listed inline in above sections as well as in complete form within
...
-- Mattr
 - Ping Identity
+- Gen Digital
+- Spruce Systems Inc.
```
Comment (Sakurann) [inline spec/spec.md:687]:
```suggestion
- Spruce Systems Inc.
- IBM
```

File: spec/spec.md [hunk 7]
```diff
@@ -682,12 +682,22 @@ Examples are listed inline in above sections as well as in complete form within
...
+- Auth0Lab
 
 ## Testing
 
-Implementations may test conformance of the wallets to this profile using this [verification website](https://vcinteroptesting.azurewebsites.net/verifier).
+Implementations may test conformance of the wallets to this profile using the following verification websites:
+
+- https://vcinteroptesting.azurewebsites.net/verifier
+- https://app.vp.interop.spruceid.xyz/
+- https://verifycredential.pingone.com/verifier.html
```
Comment (gauravping) [inline spec/spec.md:696]: For Ping, please change this to https://vc.ping-eng.com/verifier.html

Comment (Sakurann) [reply spec/spec.md:696]:
```suggestion
- https://vc.ping-eng.com/verifier.html
```

**Pull request merged**